src/python/bcc/table.py

```diff
@@ -766,3 +777,11 @@ def __len__(self):
 
     def clear(self):
         pass
+
+class DevMap(HashTable):
```

Comment:
Should DevMap be an Array instead of HashTable? The same for below CpuMap?

Reply:
Right. They should use ArrayBase.